### ISSUE TITLE
Shorthand slot props aren't recognized

### ISSUE THREAD

**Issue by jz222** [tomblachut/svelte-intellij#105]
Hi,

thanks for the work you are putting into this plugin.

I'm using [svelte-routing](https://github.com/EmilTholin/svelte-routing) in a project that makes use of slot properties, as shown in their example.

```
<Route path="/article/:id" let:params>
    <Article id="{params.id}" />
</Route>
```

While this compiles without issues, Webstorm complains that `params` on the `Article` component is not recognized.

**Comment by tomblachut** [comment]
You need to use `let:params={params}` for now :grimacing:

See #11

**Comment by tomblachut** [comment]
Released as part of 0.17.*
